application/aam-integration/README.md

```diff
@@ -0,0 +1,59 @@
+# aam-integration
+
+This application provides a third-party-integration API to authenticate external users.
```

Comment:
Does this need any environment variables whatsoever? We should add a section here to the README and probably a template env file